### DISCUSSION TITLE
CP + AD bale collection and auto field selection

### DISCUSSION THREAD

**Discussion by lordbanastor** [Courseplay/Courseplay_FS22#3197] [Q&A] [answered]
I am under an impression that this is a new issue, not an error, or maybe a feature or even a limitation, it seems, but lately I have to point manually to a field while collecting bales, or click the target icon and click on the designated field. I may be mistaken, but I could swear that ina older versions the CP could detect the field and start collecting bales from any field, with the condition of the vehicle being inside the limits of said field. Am I missing something, maybe a setting, or is this on purpose? Can this be reversed to the way it was? I mean, before, with the help of AD, I would put a checkpoint for AD inside the field and order the automatic collection and deliver of bales without any intervention. Best of regards to all.

**Comment by Tensuko** [comment, marked as answer]
That was very old but we changed it to manual so you don't need an extra AD WP on the field to start CP to collect bales.
Working as intended. Won't be changed.

Always depends on who you ask, one or the other way is better.
And NO, there won't be a setting to change that, if we put a setting in there for everything you will have to study again to be able to use CP.
